bin/lib/autovoice.js

```diff
@@ -98,6 +99,27 @@ function generatePodcast(rssUrl, voiceId=tts.defaultVoiceId){
 	;
 }
 
+function generateFirstFtBasedPodcast(maxResults, requestedUrl, includeFirstFtUuids, voiceId=tts.defaultVoiceId){
+	debug(`generateFirstFtBasedPodcast: maxResults=${maxResults}, voiceId=${voiceId}`);
+
+	return fetchContent.getLastFewFirstFtMentionedUuids(maxResults, includeFirstFtUuids)
+	.then( firstFtBasedUuids => firstFtBasedUuids.concat(individualUUIDs.list()) )
+	.then( uuids => fetchContent.articlesAsItems( uuids ) )
+	.then( items => {
+		debug(`generateFirstFtBasedPodcast: items.length=${items.length}`);
+		const promises = items.map( (item, i) => {
+			item.processingIndex = i;
+			return processItemToMp3(item, voiceId);
+		} );
+		return Promise.all(promises);
```

Comment:
None of these promises are being caught if an error occurs. All of them will be rejected if one of the set fails.